### ISSUE TITLE
predb check blowing up on title with &

### ISSUE THREAD

**Issue by TheRealShiz** [nosmokingbandit/watcher#156]
At least I assume it's the &.  I didn't dig into it.

```
INFO 2017-01-14 11:05:08,974 core.rss.predb.check_one: Checking predb.me for new available releases for A Movie with an &.
ERROR 2017-01-14 11:05:09,476 core.rss.predb.search_rss: PREDB search.
Traceback (most recent call last):
  File "C:\Watcher\watcher\core\rss\predb.py", line 80, in search_rss
    items = self.parse_predb_xml(results_xml)
  File "C:\Watcher\watcher\core\rss\predb.py", line 95, in parse_predb_xml
    root = ET.fromstring(feed)
  File "<string>", line 124, in XML
ParseError: not well-formed (invalid token): line 1, column 1548
```

Edit by NSB:
Please do not post media or release names.

**Comment by averagecdn** [comment]
Looks like I am getting something similar
ERROR 2017-01-14 14:55:40,556 core.rss.predb.search_rss: PREDB search.
Traceback (most recent call last):
  File "/Users/mark/Downloads/watcher/core/rss/predb.py", line 79, in search_rss
    results_xml = urllib2.urlopen(request).read()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 154, in urlopen
    return opener.open(url, data, timeout)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 431, in open
    response = self._open(req, data)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 449, in _open
    '_open', req)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 409, in _call_chain
    result = func(*args)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 1240, in https_open
    context=self._context)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 1197, in do_open
    raise URLError(err)
URLError: <urlopen error [SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:590)>
INFO 2017-01-14 14:55:40,557 core.sqldb.get_movie_details: Retreving details for tt1620935.
INFO 2017-01-14 14:55:40,583 core.sqldb.get_search_results: Retreving Search Results for tt1620935.
INFO 2017-01-14 14:55:40,585 core.sqldb.get_user_movies: Retreving list of user's movies.

**Comment by nosmokingbandit** [comment]
Ampersand problem fixed in d73499acab6eda953df7800bf7343b363e653b1a.

Valid XML *shouldn't* contain ampersands to begin with.
